src/makeRobot.js

```diff
@@ -38,7 +15,60 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    chipVersion: version,
+    wheels: wheels,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this.chipVersion}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `Joy: x=${this.coords.x}, y=${this.coords.y}`;
```

Comment:
Вероятно, ты хотел сделать вот так:
```suggestion
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
```

Reply:
вот это я прохытався